src/_rules/outline.js

```diff
@@ -0,0 +1,20 @@
+import { lineWidth } from '#theme';
+
+const outlineNone = {
+  outline: '2px solid transparent',
+  'outline-offset': '2px',
+};
+
+export const outlineStyle = [
+  ["outline-none", { ... outlineNone }],
```

Comment:
Apparently it works with as many spaces as we'd like here, but maybe we could change it to `...outlineNone` 😄

Reply:
https://github.com/warp-ds/drive/pull/125/commits/57b14f9397bc6a89446c78354aad71d8530c0009